Forecasting/deep4cast/models.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""Neural network models module.
+
+This module provides access to neural network models for different time
+series forecasting purposes. These models function with uni- and
+multi-variate time series alike.
+
+"""
+import keras.layers
+
+from inspect import getargspec
+from keras.models import Sequential
+
+
+class LayeredTimeSeriesModel(Sequential):
```

Comment:
Why do you need this as a class? It doesn't expose anything.

And what are other types of model?

Reply:
For example, a model with skip-connections that does not have a strict layered (or sequential structure) would be different and get it's own class. This is the case for example, for LSTNet. 

I don't think it harms us keeping the keras-inherited NN models separate from the forecaster class. It's easier to debug and more readable this way in my opinion.

Reply:
I will move the everything under forecasting to the root. Don't know what happened to the README but I will check.